packages/schema-generator/API.md

```diff
@@ -0,0 +1,52 @@
+## API Report File for "@aws-amplify/schema-generator"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public
+export class DatabaseConnectError extends Error {
+    constructor(code: string, errorno: string);
+    // (undocumented)
+    readonly code: string;
+    // (undocumented)
+    readonly errorno: string;
+}
+
+// @public (undocumented)
+export type MySQLEngine = 'mysql';

```

Comment:
```suggestion
export type MySqlEngine = 'mysql';

```

Please apply in other places too if you take this.

Reply:
Updated.